tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -146,38 +154,29 @@ def parse_args():
         os.path.basename(__file__),
         description='''Convert the ONNX format model/s in the provided directory to ORT format models.
         All files with a `.onnx` extension will be processed. For each one, an ORT format model will be created in the
-        same directory. A configuration file will also be created called `required_operators.config`, and will contain
+        same directory. If supported by the optimization style, a configuration file will also be created containing
         the list of required operators for all converted models.
```

Comment:
When would we not generate a configuration file with the required operators?

Reply:
now it is skipped if runtime optimizations are enabled because the config file may be incomplete.

Reply:
If we need to diff the two config files to figure out the runtime optimization kernels to include we'll need it to generate it. Also that would mean you'd only get the config for the fully optimized model, which may be missing kernels that the runtime optimization version needs. e.g. if the fully optimized version includes optimizations that are not yet supported via runtime optimization, the pre-optimization kernels would be needed but wouldn't be in the config file

Reply:
added another step to also convert models without runtime optimizations and add those to the models used to create the config file.